Changes to curl test 172 to make it work okay.
This simply uses the latest version of the test at:
https://raw.githubusercontent.com/bagder/curl/master/tests/data/test172

This change will not be sent upstream and this patch can be removed when
we upgrade to the latest version of curl.

--- tests/data/test172.orig	2014-09-09 08:41:51.039292198 -0700
+++ tests/data/test172	2014-09-09 08:41:50.918034165 -0700
@@ -31,12 +31,12 @@
 </command>
 <file name="log/jar172.txt">
 # Netscape HTTP Cookie File
-# http://curl.haxx.se/rfc/cookie_spec.html
+# http://curl.haxx.se/docs/http-cookies.html
 # This file was generated by libcurl! Edit at your own risk.
 
 .%HOSTIP	TRUE	/silly/	FALSE	0	ismatch	this
 .%HOSTIP	TRUE	/	FALSE	0	partmatch	present
-%HOSTIP	FALSE	/we/want/	FALSE	1391252187	nodomain	value
+%HOSTIP	FALSE	/we/want/	FALSE	2139150993	nodomain	value
 </file>
 </client>
 
